process/naming.md

```diff
@@ -70,3 +70,7 @@ Examples of release tags:
 
 It's quite easy to understand which branch was used for the release and
 to find all releases for your branch.
+
+Release tags can't be moved after creation. If fixes/updates are made to the
+release, then new release tag should be created with the increment of the
```

Comment:
not exactly. Update may change public API, so major part of the tag should be increased. Jus say something like

"new release tag should be created according to release rules and SemVer requirements"